recipes/pthread-win32/all/conanfile.py

```diff
@@ -32,7 +32,7 @@ def build(self):
                              12: 'pthread.2013.sln'}.get(int(str(self.settings.compiler.version)))
             targets = ['pthread_dll'] if self.options.shared else ['pthread_lib']
             msbuild = MSBuild(self)
-            msbuild.build(solution_name, targets=targets, platforms={"x86": "Win32"})
+            msbuild.build(solution_name, targets=targets, upgrade_project=False, platforms={"x86": "Win32"})
```

Comment:
Thanks, I did not know this option nor what it does!

Reply:
it updates the msvc project file to the current visual version. but as you should know, visual studio breaks when trying to update an old project to a new ide version. At least, that's my impression from years haha

Reply:
On the other hand upgrades are often required....

I had major pain which projects  which use premake4. The newest VS version their supporting is VS2012

Reply:
that's true, but is a trade-off situation. maybe we can provide an env var force it when installing/building a package.